src/io/flutter/preview/DartElementPresentationUtil.java

```diff
@@ -88,18 +89,23 @@ public static Icon getIcon(@NotNull Element element) {
     }
   }
 
-  @NotNull
-  public static String getText(@NotNull Element element) {
-    final StringBuilder b = new StringBuilder(element.getName());
+  public static void renderElement(@NotNull Element element, @NotNull OutlineTreeCellRenderer renderer, boolean nameInBold) {
+    final SimpleTextAttributes attribs =
```

Comment:
Can we use full names, e.g. `attributes`?

Reply:
sgtm